### ISSUE TITLE
CI: Error while executing topic command : replication factor: 1 larger than available brokers: 0

### ISSUE THREAD

**Issue by joestringer** [cilium/cilium#4505]
I hit this in a PR:
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/4132

#### runtime.RuntimeValidatedKafka Kafka Policy Ingress (from (runtime.xml))
```
Failing for the past 1 build (Since ￼#4132 )
Took 23 sec.
```
#### Stacktrace
```
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:330
Unable to create topic  allowedTopic
Expected command: docker exec -i  client /opt/kafka/bin/kafka-topics.sh --create --zookeeper zook:2181 --replication-factor 1 --partitions 1 --topic allowedTopic 
To succeed, but it failed:
Exitcode: 1 
Stdout:
 	 Error while executing topic command : replication factor: 1 larger than available brokers: 0
	 
Stderr:
 	 [2018-06-13 18:14:30,008] ERROR org.apache.kafka.common.errors.InvalidReplicationFactorException: replication factor: 1 larger than available brokers: 0
	  (kafka.admin.TopicCommand$)
	 

/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/runtime/kafka.go:82
```
#### Standard Output
```
STEP: Creating kafka topics
[[ATTACHMENT|34040e1e_RuntimeValidatedKafka_Kafka_Policy_Ingress.zip]]
```

Unfortunately the zip doesn't contain much detail, seems it failed in `BeforeEach()`:

https://github.com/cilium/cilium/blob/fcb01193c2fd21313a8262c1c09105b39a4e8b15/test/runtime/kafka.go#L113-L122

https://github.com/cilium/cilium/blob/fcb01193c2fd21313a8262c1c09105b39a4e8b15/test/runtime/kafka.go#L77-L82

[test-output.log](https://github.com/cilium/cilium/files/2099634/test-output.log)

**Comment by ianvernon** [comment]
Filed #4506 since no logs were gathered.

**Comment by eloycoto** [comment]
Rebase please, should be fixed with my changes in microscope.

**Comment by ianvernon** [comment]
@eloycoto , it looks that commit is in @joestringer 's branch: 964321bcf0a2520bc862f0ada1596a2dd8ee1302 

Joe's branch: https://github.com/joestringer/cilium/commits/a2186408e2cf02648267490ad1bfe5f6589a26e4?after=a2186408e2cf02648267490ad1bfe5f6589a26e4+34

**Comment by joestringer** [comment]
Minor link correction: https://github.com/joestringer/cilium/commits/5a1b860baf4d48afca5574778938434cc5eab2c8?after=a2186408e2cf02648267490ad1bfe5f6589a26e4+34 but the point remains, 964321b is in that branch.

**Comment by ianvernon** [comment]
Leaving this open in case it gets hit again. Given that we had no logs previously, will close if this doesn't occur again.

**Comment by manalibhutiyani** [comment]
Should be fixed by #4488

**Comment by manalibhutiyani** [comment]
Please reopen if you see this again
